docs/kyma/01-01-in-a-nutshell.md

```diff
@@ -3,14 +3,18 @@ title: In a nutshell
 type: Overview
 ---
 
-Kyma allows you to connect applications and third-party services in a cloud-native environment. Use it to create extensions for the existing systems, regardless of the language they are written in. Customize extensions with minimum effort and time devoted to learning their configuration details.
+Kyma allows you to extend applications with microservices and lambda functions. First, connect your application to a Kubernetes cluster and expose the application's API or Events securely. Then, implement the business logic you require by creating microservices or functions, and triggering them to react to particular Events or calls to your application's API. To limit the time spent on coding, use the in-built cloud services from the Service Catalog, exposed by open service brokers from such cloud providers as GCP, Azure, and AWS.
 
-Focus purely on coding with these out-of-the-box functionalities at hand:
-- Service-to-service communication and proxying (Istio Service Mesh)
-- In-built monitoring, tracing, and logging (Grafana, Prometheus, Jaeger, Loki, Kiali)
-- Secure authentication and authorization (Dex, Service Identity, TLS, Role Based Access Control)
-- The catalog of services to choose from (Service Catalog, Service Brokers)
-- The development platform to run lightweight functions in a cost-efficient and scalable way (Serverless, Kubeless)
-- The endpoint to register Events and APIs of external applications (Application Connector)
-- The messaging channel to receive Events, enrich them, and trigger business flows using lambdas or services (Event Bus, NATS)
-- CLI supported by the intuitive UI (Console)
+Kyma comes equipped with these out-of-the-box functionalities:
```

Comment:
I think I saw it somewhere already :)

makes sense to have it here, we just need to make sure all TWs know about it, like about Key features and Components docs, so this sections are always up to date. Like for example the API gateway link in future once new gateway is introduced. Same thing with compass of knative functions

Reply:
I will let all know :)